index.js

```diff
@@ -199,6 +199,7 @@ const plugin = fp(async function (app, opts) {
       errorHandler: opts.errorHandler,
       errorFormatter: opts.errorFormatter,
       ide: optsIde,
+      ideSettings: opts.playgroundOpts,
```

Comment:
Could you please add the option to the README? I would call it `playgroundSettings` as it's not available for graphiql.

Reply:
okay